### ISSUE TITLE
Missing default permissions in Getting Started with Snowflake - Zero to Snowflake

### ISSUE THREAD

**Issue by imnicmarshall** [Snowflake-Labs/sfquickstarts#641]
Getting Started with Snowflake - Zero to Snowflake -> Section 3. The Snowflake ​User Interface & Lab Story

Under the subheading "warehouse" it states the the sysadmin role has access to the Compute_WH by default. However, this was not the case for me.

I'm unsure if this should be fixed on the backend or the documentation should include a step for adding the permission. I went and added my own permission and am happy to submit a request for the edit to the documentation.

https://quickstarts.snowflake.com/guide/getting_started_with_snowflake/index.html#2

![image](https://user-images.githubusercontent.com/17224344/235505911-a169049c-ad92-4bee-924d-68cd7f54ea31.png)

**Comment by bmccormi99** [comment]
I agree that this is an error in the Zero to Snowflake document.  The COMPUTE_WH was created under the ACCOUNTADMIN id and not sysamdin.  Instead of adding a permission to sysadmin, I just used ACCOUNTADMIN for the rest of the document.  It did take me a bit to find this once the screen shot and instruction didn't match the actual screens ("Select the following context settings:  Role: SYSADMIN Warehouse: COMPUTE_WH").

**Comment by gman9093** [comment]
If you want to follow the documentation as is, you can simply grant SYSADMIN usage on the warehouse:
   GRANT USAGE ON WAREHOUSE COMPUTE_WH TO ROLE SYSADMIN;
This allows your user (the one created during setup) to execute commands using the warehouse against the database you just created.  Obviously, this is not what you'd do in a real environment but this is a demo.
